eosiojava/src/main/java/one/block/eosiojava/interfaces/ISerializationProvider.java

```diff
@@ -76,4 +79,24 @@
      * conversion process.
      */
     String serializeAbi(String json) throws SerializeAbiError;
+
+    /**
+     * Convenience method to transform hex string with context free data to an array of data.
+     *
+     * @param hex Hex string representing the context free data to deserialize.
+     * @return Deserialized String representing the data hex.
+     * @throws DeserializeContextFreeDataError A deserialization error is thrown if there are any exceptions during the
+     * conversion process.
+     */
+    List<String> deserializeContextFreeData(String hex) throws DeserializeContextFreeDataError;
+
+    /**
+     * Convenience method to transform an array of data to a hex string.
+     *
+     * @param contextFreeData Array of data to serialize.
+     * @return Serialized hex string representing the array of data.
+     * @throws SerializeContextFreeDataError A serialization error is thrown if there are any exceptions during the
+     * conversion process.
+     */
+    String serializeContextFreeData(List<String> contextFreeData) throws SerializeContextFreeDataError;
```

Comment:
The context free data is not being serialized by the ABIEOS Serialization provider so this task should probably be given to the TransactionProcessor.  Again, let's discuss this approach.